client/containers/VotingContainer.ts

```diff
@@ -158,6 +158,26 @@ export default class VotingContainer extends Container<VotingContainerState> {
     );
   }
 
+  /**
+   * Deletes a prevote. If a prevote is deleted, the final vote it deleted as well.
+   * @param companyId The company that is currently being voted on.
+   * @param partnerId The id of the voting partner.
+   */
+  async deletePrevote(companyId: number, partnerId: number) {
+    const company = this.company(companyId);
+    if (!company) return;
+
+    const prevoteObj = company.partnerVotes.prevote.find(
```

Comment:
Wondering if line 170 happens asynchronously? Does this create some kind of race condition with slower response times to the backend (i.e. in production).

Reply:
good catch